src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -113,7 +113,7 @@
 helps['keyvault create'] = """
 type: command
 short-summary: Create a key vault.
-long-summary: Default permissions are created for the current user or service principal unless the `--no-self-perms` flag is specified.
+long-summary: If `--enable-rbac-authorization` is not specified, then default permissions are created for the current user or service principal unless the `--no-self-perms` flag is specified.
```

Comment:
This statement is cumbersome.

```suggestion
long-summary: Default permissions are created for the current user or service principal unless the `--no-self-perms` or `--enable-rbac-authorization` flag is specified.
```